### DISCUSSION TITLE
Monitoring cloud service status

### DISCUSSION THREAD

**Discussion by garrmark** [ravendb/ravendb#14837] [Ideas]
There is a DNS issue on Azure today that affected the ravendb cloud service for us and the https://status.ravendb.net/ page showed the outage and resolution steps.

Is there a good way for us to monitor this page or a similar API that would return a non 200 status code when there are issues reported?

Maybe:
https://status.ravendb.net/status/api
https://status.ravendb.net/status/cloud-portal
.
.
.

 could return some 400-500 codes when there are outages, etc

**Comment by ayende** [comment]
That is a good idea, yes.

**Comment by gregolsky** [comment]
Thanks! I created an issue in our internal bugtracker for that.

**Comment by garrmark** [comment]
is this still in the works @ayende @gregolsky ?

**Reply by ayende** [reply]
Hi, this is on our roadmap and we hope to have that by the end of Feb

**Comment by gregolsky** [comment]
Hi @garrmark,

The endpoint it https://status.ravendb.net/api/v1/service/status
It returns the following response status codes:
- 200 - no issues
- 550 - anything happening, including regular service maintenance 

If you have any questions please let us know.

**Reply by garrmark** [reply]
Is this an aggregation of all items on the https://status.ravendb.net page?

**Reply by gregolsky** [reply]
Yes, at the moment if anything on the status page is out of the ordinary you will get a 550. 200 otherwise.
